src/Forms/ElementalAreaField.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace DNADesign\Elemental\Forms;
+
+use SilverStripe\Forms\GridField\GridField;
+use SilverStripe\Forms\CompositeField;
+use SilverStripe\Forms\FieldGroup;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\TabSet;
+use SilverStripe\Forms\ReadonlyTransformation;
+
+class ElementalAreaField extends GridField
+{
+    /**
+     * A getter method that seems redundant in that it is a function that returns a function,
+     * however the returned closure is used in an array map function to return a complete FieldList
+     * representing a read only view of the element passed in (to the closure).
+     *
+     * @return function
```

Comment:
I don't think `function` is a valid PHPDoc type - maybe use `callable` or `Closure`